labs/3.md

```diff
@@ -39,7 +39,7 @@ You can access to the Argo CD WebUI using your browser and the hostname returned
 We won't be using the WebUI until Lab10, feel free to access the WebUI at any time to follow the application deployments.
 
 <a id="markdown-how-to-login-to-Argo-CD-cli" name="how-to-login-to-Argo-CD-cli"></a>
-## How-to Login to Argo CD CLI
+## How-to Login to Argo CD CLI (For reference only)
```

Comment:
I like the emphasis here.  I think that we also need to call that out in the slides as we are warming people up to the lab.